pkg/deviceshifu/deviceshifulwm2m/deviceshifulwm2m.go

```diff
@@ -0,0 +1,244 @@
+			}
+
+			respString = "Success"
+		case http.MethodGet:
+			data, err := handlerServer.Read(objectId)
+			if err != nil {
+				http.Error(w, "Error on reading object", http.StatusBadRequest)
+				logger.Errorf("Error on reading object" + err.Error())
+				return
+			}
+			respString = data
+		case http.MethodPost:
+			requestBody, err := io.ReadAll(r.Body)
+			if err != nil {
+				http.Error(w, "Error on parsing body", http.StatusBadRequest)
+				logger.Errorf("Error on parsing body" + err.Error())
+				return
+			}
+
+			err = handlerServer.Execute(objectId, string(requestBody))
+			if err != nil {
+				http.Error(w, "Error on executing object", http.StatusBadRequest)
+				logger.Errorf("Error on executing object" + err.Error())
+				return
+			}
+
+			respString = "Success"
+		default:
+			http.Error(w, "request method not support", http.StatusMethodNotAllowed)
+			logger.Errorf("Request method %s is not support!", r.Method)
+			return
+		}
+
+		instructionFuncName, shouldUsePythonCustomProcessing := deviceshifubase.CustomInstructionsPython[handlerInstruction]
+		logger.Infof("Instruction %v is custom: %v", handlerInstruction, shouldUsePythonCustomProcessing)
+		if shouldUsePythonCustomProcessing {
+			logger.Infof("Instruction %v has a python customized handler configured.\n", handlerInstruction)
+			respString = utils.ProcessInstruction(deviceshifubase.PythonHandlersModuleName, instructionFuncName, respString, deviceshifubase.PythonScriptDir)
+		}
+		fmt.Fprintf(w, "%v", respString)
+	}
+}
+
+func (ds *DeviceShifuLwM2M) collectHTTPTelemtries() (bool, error) {
+	if ds.server.Conn == nil {
+		return false, nil
+	}
+
+	if err := ds.server.Conn.Ping(ds.server.Conn.Context()); err != nil {
+		logger.Errorf("Error checking telemetry: %v", err.Error())
+		return false, err
+	}
+
+	if ds.base.EdgeDevice.Spec.Protocol != nil {
+		switch protocol := *ds.base.EdgeDevice.Spec.Protocol; protocol {
+		case v1alpha1.ProtocolLwM2M:
+			telemetries := ds.base.DeviceShifuConfig.Telemetries.DeviceShifuTelemetries
+			instructions := ds.base.DeviceShifuConfig.Instructions
+			for telemetry, telemetryProperties := range telemetries {
+				if ds.base.EdgeDevice.Spec.Address == nil {
+					return false, fmt.Errorf("Device %v does not have an address", ds.base.Name)
+				}
+
+				instructionName := telemetryProperties.DeviceShifuTelemetryProperties.DeviceInstructionName
+				if instructionName == nil {
+					return false, fmt.Errorf("Device %v telemetry %v does not have an instruction name", ds.base.Name, telemetry)
+				}
+
+				if instructions.Instructions[*instructionName] == nil {
+					return false, fmt.Errorf("Device %v telemetry %v instruction %v does not exist", ds.base.Name, telemetry, *instructionName)
+				}
+
+				objectId := instructions.Instructions[*instructionName].DeviceShifuProtocolProperties["ObjectId"]
+				if objectId == "" {
+					return false, fmt.Errorf("Device %v telemetry %v does not have an object id", ds.base.Name, telemetry)
+				}
```

Comment:
do we need to explict lwm2m in all log text here, if we have to, please add lwm2m in all logs